Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml

```diff
@@ -285,7 +285,7 @@
   - type: StaticPrice
     price: 125
   - type: EntityStorage
-    airtight: false
+    capacity: 5000
```

Comment:
This is the capacity for entities, not air. You would need to make Air in EntityStorageComponent a data field and modify its volume in the yaml.

Reply:
waz